planning/behavior_path_planner/src/scene_module/goal_planner/goal_planner_module.cpp

```diff
@@ -665,29 +665,27 @@ void GoalPlannerModule::setLanes()
 
 void GoalPlannerModule::setOutput(BehaviorModuleOutput & output)
 {
-  if (status_.is_safe_static_objects) {
-    if (isSafePath()) {
-      // clear stop pose when the path is safe against static/dynamic objects and activated
-      if (isActivated()) {
-        resetWallPoses();
-      }
-      // keep stop if not enough time passed,
-      // because it takes time for the trajectory to be reflected
-      auto current_path = getCurrentPath();
-      keepStoppedWithCurrentPath(current_path);
-
-      output.path = std::make_shared<PathWithLaneId>(current_path);
-      output.reference_path = getPreviousModuleOutput().reference_path;
-    } else if (status_.has_decided_path && isActivated()) {
-      // situation : not safe against dynamic objects after approval
-      // insert stop point in current path if ego is able to stop with acceleration and jerk
-      // constraints
-      setStopPathFromCurrentPath(output);
+  if (!status_.is_safe_static_objects) {
+    // situation : not safe against static objects use stop_path
+    setStopPath(output);
+  } else if (!isSafePath() && status_.has_decided_path && isActivated()) {
```

Comment:
just question
not just use `status_.is_safe_dynamic_objects` but need to calculate isSafePath()?

and I did not find where `status_.is_safe_dynamic_objects` is set.

Reply:
@kosuke55 
Thank you so much for confirmation.
I failed to set the variable true.
changed in https://github.com/autowarefoundation/autoware.universe/pull/5007/commits/8f81b7f476c2400a3dd5dbf381d8a0de3bac47e9

Reply:
@kyoichi-sugahara 
!isSafePath() but set `status_.is_safe_dynamic_objects = true;` :thinking:

Reply:
@kosuke55 
deleted `status_.is_safe_dynamic_objects` :angry: in https://github.com/autowarefoundation/autoware.universe/pull/5007/commits/080e59542d46bc2842797f2124ab7a0a41c23f80